src/app/options.cpp

```diff
@@ -247,6 +315,13 @@ namespace
     constexpr const StringOption CONFIGURATION_OPTION = {"configuration"};
     constexpr const BoolOption PORTABLE_OPTION = {"portable"};
     constexpr const BoolOption RELATIVE_FASTRESUME = {"relative-fastresume"};
+    constexpr const StringOption SAVE_PATH_OPTION = {"save-path"};
+    constexpr const TriStateBoolOption PAUSED_OPTION = {"add-paused"};
+    constexpr const BoolOption SKIP_HASH_CHECK_OPTION = {"skip-hash-check"};
+    constexpr const StringOption CATEGORY_OPTION = {"category"};
+    constexpr const BoolOption SEQUENTIAL_OPTION = {"sequential"};
+    constexpr const BoolOption FIRST_AND_LAST_OPTION = {"first-and-last"};
```

Comment:
It's not a bug. We can't prioritize something since we haven't any torrent metadata in case we add it using magnet link. We can do it deferred when metadata has to be loaded. Please leave TODO comment near corresponding AddTorrentParams member.